test/runtests.jl

```diff
@@ -1,23 +1,24 @@
 using Dispatcher
 using ResultTypes
-using Base.Test
+using Compat.Test
 using Memento
 using IterTools
-
+using Compat.Distributed
+using DeferredFutures
 import LightGraphs
 
 using Compat: @__MODULE__
 using ResultTypes: iserror
 
 const LOG_LEVEL = "info"      # could also be "debug", "notice", "warn", etc
 
-Memento.config(LOG_LEVEL; fmt="[{level} | {name}]: {msg}")
+Memento.config!(LOG_LEVEL; fmt="[{level} | {name}]: {msg}")
 const logger = getlogger(@__MODULE__)
 
 function test_addproc(x::Int; level=LOG_LEVEL)
     ret = addproc(x)
-    @everywhere using Dispatcher
-    @everywhere using Memento
+    @everywhere @eval using Dispatcher
+    @everywhere @eval using Memento
     @everywhere Memento.config(level; fmt="[{level} | {name}]: {msg}")
```

Comment:
Use `config!`

Reply:
You can also remove the `fmt` argument here